index.html

```diff
@@ -52,13 +52,31 @@ <h1 class="header-title">
         <div class="application-demo__container">
           <img
             class="application-demo__img"
-            src="/images/Group 53 (1).png"
+            src="images/Group 53 (1).png"
             alt="application demo"
           />
           <hr class="application-demo__line" />
         </div>
       </div>
     </section>
-    <section>lcnj;sbc;aiygc;iy</section>
+    <section class="description">
+      <div class="container">
+        <h1 class="description-text">How the app works</h1>
```

Comment:
It's not separate sections, let's put it inside first description-section above description-section__row and remove section + container tags.
Also lets change description-text to 

```suggestion
        <h1 class="description-section__main-title">How the app works</h1>
```